docs/level-2.md

```diff
@@ -19,7 +19,7 @@ import EarlyGameQuestion4 from '@site/image-generator/yml/beginner/early-game-qu
 
 <br />
 
-## Special Moves
+## Stalls and Finesses
```

Comment:
special moves is a special header that every single level has, why rename this?

Reply:
It was in my original proposal and you didn't object.

Reply:
i didnt understand it properly I guess.
every single level should have header called "Special Moves" for uniformity.

Reply:
No problem at all, resending